editor/editor_help.cpp

```diff
@@ -45,6 +47,33 @@
 
 DocTools *EditorHelp::doc = nullptr;
 
+class DocCache : public Resource {
+	GDCLASS(DocCache, Resource);
+	RES_BASE_EXTENSION("doc_cache");
+
+	String version_hash;
+	Array classes;
+
+protected:
+	static void _bind_methods() {
+		ClassDB::bind_method(D_METHOD("set_version_hash", "version_hash"), &DocCache::set_version_hash);
+		ClassDB::bind_method(D_METHOD("get_version_hash"), &DocCache::get_version_hash);
+
+		ClassDB::bind_method(D_METHOD("set_classes", "classes"), &DocCache::set_classes);
+		ClassDB::bind_method(D_METHOD("get_classes"), &DocCache::get_classes);
+
+		ADD_PROPERTY(PropertyInfo(Variant::STRING, "version_hash"), "set_version_hash", "get_version_hash");
+		ADD_PROPERTY(PropertyInfo(Variant::DICTIONARY, "classes"), "set_classes", "get_classes");
```

Comment:
PropertyInfo of property `classes` here uses the type **Dictionary** but the argument of setter `DocCache::set_classes` and the return type of getter `DocCache::get_classes` is **Array**. So there is a mismatch.

Reply:
Good catch, my friend. Fixed in #77782.